frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -78,11 +81,24 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
   const vmiLike = kindObj === VirtualMachineModel ? vm : vmi;
   const vmServicesData = getServicesForVmi(getLoadedData(props.services, []), vmi);
   const canUpdate = useAccessReview(asAccessReview(kindObj, vmiLike || {}, 'patch')) && !!vmiLike;
+  const [guestAgentInfo] = useGuestAgentInfo({ vmi });
+
+  const OSMismatch =
+    vmi && guestAgentInfo && isWindows(vmiLike) !== (guestAgentInfo?.os?.id === 'mswindows');
+  const OSMismatchAlert = OSMismatch && (
+    <Alert className="co-alert" variant="warning" title="Operating syatem mismatch" isInline>
```

Comment:
There's a typo in the title: syatem => system

Reply:
Thanks :exclamation: